### PR TITLE
add SVG classList support for IE

### PR DESCRIPTION
**Pull request by jesper-johansson**

### CODE REVIEW

File: tests/dom.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ QUnit.test("Helpers - getClassList() and getRelList()", function(assert) {
   assert.ok(getClassList(document.getElementById('baz')).contains('epsilon'));
   assert.ok(getClassList(document.getElementById('baz')).contains('phi'));
 
-  var elem = document.createElement('span');
+  var elem = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
```
Comment (inexorabletash) [inline tests/dom.js]: I suspect you meant to add cases for this, rather than replacing the existing cases.

File: dom.js [hunk 2]
```diff
@@ -418,7 +418,7 @@
 
     // HTML - https://html.spec.whatwg.org
     // Element.classList
-    if ('classList' in document.createElement('span')) {
+    if ('classList' in document.createElementNS('http://www.w3.org/2000/svg', 'svg')) {
```
Comment (inexorabletash) [inline dom.js]: This line is trying to answer: "when installing a helper that can be used even in IE7-, is the classList property available on elements, or does a DOMTokenListShim need to be created explicitly?"

This line would only need to be changed if there was a version of IE7- that supported the classList property on span but not on svg, which is not the case.

Comment (jesper-johansson) [reply dom.js]: I'm not sure I understand. `.classList` doesn't work on SVG elements in higher versions of IE than 7, so wouldn't you'd want to make the check on an SVG element?

Comment (inexorabletash) [reply dom.js]: In IE8 and higher the `classList` getter polyfill can be added to `Element prototype` (see later code) so the `window.getClassList` helper can just call `elem.classList`

**Pull request closed without merging**